tools/wptrunner/wptrunner/formatters/chromium.py

```diff
@@ -272,5 +272,6 @@ def suite_end(self, data):
         return json.dumps(final_result)
 
     def process_output(self, data):
-        if 'command' in data and 'chromedriver' in data['command']:
+        cmd = data.get('command', "")
```

Comment:
```suggestion
        cmd = data.get("command", "")
```

Nit: Don't mix `'` and `"` in the same line. (Preferably not in the same file, but I acknowledge that this file already has mismatching uses!).